app/src/main/java/com/vci/vectorcamapp/incomplete_session/presentation/IncompleteSessionScreen.kt

```diff
@@ -41,6 +51,38 @@ fun IncompleteSessionScreen(
         },
         modifier = modifier
     ) {
+        item {
+            TextEntryField(
```

Comment:
Now that this search bar is being used in multiple files, you should abstract it out into a common component used throughout the app. Create a search package within core/presentation, and create a new component for search bar.